Automattic/wp-calypso
# Calculate the minimum amount based on the currency

#### Changes proposed in this Pull Request

* Add currency minimums for supported Stripe currencies
* Enforce minimum price based on currency

#### Testing instructions

* With a "Premium Plan" site:
  - Go to "My Sites → Tools Earn"
  - Click on "Collect Recurring Payments" button in the "Recurring Payments" card
  - Under "Settings" click on "Recurring Payment Plans >"
  - Click "Add new plan" _or_ "Edit" from the ellipsis menu of an existing plan.
  - For the given currency set a price below the minimum amount:

     ```
	USD: 0.5
	AUD: 0.5
	BRL: 0.5
	CAD: 0.5
	CHF: 0.5
	DKK: 2.5
	EUR: 0.5
	GBP: 0.3
	HKD: 4.0
	INR: 0.5
	JPY: 50
	MXN: 10
	NOK: 3.0,
	NZD: 0.5
	PLN: 2.0
	SEK: 3.0
	SGD: 0.5
     ```

You will see an error informing you to use a value higher than the minimum amount:

<img width="718" alt="image" src="https://user-images.githubusercontent.com/19795/75589813-a6b48580-5a30-11ea-949c-a28fa9451e88.png">

When changing the currency _or_ the price, the error should reflect the wether the amount provided meets the minimum requirement.

See Automattic/jetpack#14802
